documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,141 @@
+      1. Qualification for an event must be based only on results from the same event.
+   2. If the competitor limit for a competition or an event is reached before the competition, the organization team must not approve any additional registrations for the competition or the event.
+   3. The competitor limit is per competition. Per day competitor limits are not allowed.
+6. Registration requirements
+   1. The registration requirements must be clear so that any competitor from anywhere in the world knows how to and has the ability to register.
+      1. The registration requirements must be available in English.
+      2. In addition to the English version, the registration requirements can be available in any other language and should be available in the local language.
+   2. The registration period(s) must be a clearly defined time range with a start and end time and date.
+      1. The first registration period must have a start time and date at least 48 hours after the competition is announced.
+         1. The competition's Delegate(s) and/or organizers may register for the competition prior to the first registration period.
+      2. If there is more than one registration period, the organization team must announce each additional registration period at least 48 hours before it starts.
+      3. Different registration periods may have different registration requirements (e.g. restricting which events a competitor can register for).
+   3. Different locations for multi-location competitions may have different registration requirements (e.g. competitor limit).
+7. Events
+   1. Each event must have a planned number of rounds.
+      1. Each round of each event must have a planned format and time limit.
+      2. A round of an event must have planned advancement conditions if there is a subsequent round of the event.
+      3. A round may have a cutoff. A round with a cutoff is a combined round, with the formats as per [WCA Regulation 9b)](wca{regulations/#9b}).
+   2. Only results from events that appear on the competition website in the Events tab and Schedule will be recognized.
+      1. Other events may only be held unofficially.
+8. Schedule
+   1. The schedule must be published on the WCA website using the WCA schedule tool.
+9. Championships
+   1. A championship type is assigned to competitions that are National, Continental, or World Championships.
+      1. A competition must be located in the region where the championship type is selected.
+      2. A competition may have multiple championship types.
+      3. A championship type for a region must only be assigned to one competition in a year.
+      4. An additional selection process is conducted by the WCA Board for organizing Continental and World Championships.
+   3. Naming
+      1. Continental and World Championships names must begin with "WCA".
+      2. The word "Championship" must be singular in the competition name.
```

Comment:
```suggestion
      2. If used, the word "Championship" must be singular in the competition name.
```

Otherwise, German Nationals and German Open would not be acceptable names for a championship.